packages/fmg-turbo-adjudicator/contracts/TurboAdjudicator.sol

```diff
@@ -73,16 +62,16 @@ contract TurboAdjudicator {
         );
 
         Authorization memory authorization = Authorization(
+            participant,
             destination,
             amount,
-            withdrawalNonce[participant]
+            msg.sender
         );
         require(
             keccak256(encodedAuthorization) == keccak256(abi.encode(authorization)),
```

Comment:
Given that you have to calculate it here anyway, do you get anything from passing the `encodedAuthorization` through in the method parameters?

Reply:
I guess you save a little gas, but that's it. I think I'll change this